test/basic/init_funcs.c

```diff
@@ -0,0 +1,21 @@
+#include "smack.h"
+
+// @expect verified
+// @flag --unroll=1
+
+void __SMACK_INIT(defineProcessStates) {
+  __SMACK_top_decl("const unique $process_uninitialized: int;");
```

Comment:
I see, fair enough, makes sense... Maybe you could create two examples - mine and yours :).
Also, please change your _fail version into something that resembles your not failing version. If you look at other regressions, that is how we typically do it. So maybe copy-paste more or less the same thing, and just change the final assertions into something that should fail.